app/src/main/java/com/kodeco/android/countryinfo/flow/Flows.kt

```diff
@@ -0,0 +1,46 @@
+package com.kodeco.android.countryinfo.flow
+
+import kotlinx.coroutines.DelicateCoroutinesApi
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+
+@OptIn(DelicateCoroutinesApi::class)
+object Flows {
+
+    val tapFlow: StateFlow<Int>
+        get() = _tapFlow
+
+    val backFlow: StateFlow<Int>
+        get() = _backFlow
+
+    val counterFlow: StateFlow<Int>
+        get() = _counterFlow
+
+    private val _tapFlow = MutableStateFlow(0)
+    private val _backFlow = MutableStateFlow(0)
+    private val _counterFlow = MutableStateFlow(0)
+
+    init {
+
+        GlobalScope.launch {
+            var timer = 0
+            while (true){
+                _counterFlow.value = timer++
+                delay(1000)
```

Comment:
really minor here but since the initial value of `_counterFlow` is already 0 you could swap this around a bit:

```suggestion
                delay(1000)
                _counterFlow.value = ++timer
```

Putting the incrementer `++` before `timer` will auto-increment it by 1 first, then set the value of `_counterFlow`.  The way it is right now, you have the initial value of `_counterFlow` at 0, then because `++` comes _after_ timer, the value of `_counterFlow` gets set again to 0 and then `timer` gets incremented by 1.